docker-compose.yml

```diff
@@ -277,6 +277,22 @@ services:
       - _APP_SMTP_USERNAME
       - _APP_SMTP_PASSWORD
 
+  appwrite-maintenance:
+    entrypoint: maintenance
+    container_name: appwrite-maintenance
+    build:
+      context: .
+    restart: unless-stopped
+    networks:
+      - appwrite
+    depends_on:
+      - redis
+      - mariadb
```

Comment:
I think you don't need MariaDB here, if you do make sure to pass DB env vars as well as the Redis ones.

Reply:
Hmm.. I need to be making database calls to get the list of project IDs right?

Reply:
Yes. no problem here.